docs/installing.md

```diff
@@ -1,4 +1,14 @@
-# OS X fast track
+
+# Operating systems
+
+gcsfuse has been tested successfully with the following operating systems:
+
+*   Linux (minimum kernel version 3.10)
+*   OS X (minimum version 10.10.2)
+
+It may or may not work correctly with other operating systems and older versions.
+___
```

Comment:
Remove the line here? It looks odd, especially if the thing below is supposed to be a sub-section.